File: sub-web-model/src/main/java/org/github/base/service/impl/ArticleServiceImpl.java
package org.github.base.service.impl;

import org.github.base.entity.ArticleEntity;
import org.github.base.mapper.IArticleMapper;
import org.github.base.service.IArticleService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 * <p>
 *  服务实现类
 * </p>
 *
 * @author JYD_XL
 * @since 2021-05-08
 */
@Service
public class ArticleServiceImpl extends ServiceImpl<IArticleMapper, ArticleEntity> implements IArticleService {}

File: sub-mysql-blog-service/src/main/java/org/github/mysql/blog/base/service/impl/ArticleServiceImpl.kt
@file:Suppress("RedundantSemicolon")

package org.github.mysql.blog.base.service.impl;

import org.github.mysql.blog.base.entity.ArticleEntity;
import org.github.mysql.blog.base.mapper.IArticleMapper;
import org.github.mysql.blog.base.service.IArticleService;
import org.github.base.ServiceImpl;
import org.springframework.stereotype.Service;

/**
 *  服务实现类
 *
 * @author JYD_XL
 */
@Service
open class ArticleServiceImpl : ServiceImpl<IArticleMapper, ArticleEntity>(), IArticleService
